modules/core/src/main/scala/advxml/instances/AllTransforInstances.scala

```diff
@@ -96,7 +96,8 @@ private[instances] trait XmlModifierInstances {
       * @param d Attribute data.
       * @param ds Attributes data.
       */
-    def apply(d: AttributeData, ds: AttributeData*): SetAttrs = SetAttrs(NonEmptyList.of(d, ds: _*))
+    def apply(d: AttributeData, ds: AttributeData*): SetAttrs = SetAttrs(_ => NonEmptyList.of(d, ds: _*))
```

Comment:
Another user-friendly apply method is needed that takes a function Map[String, String] => ??? as argument. Don't know though what ??? should be.